src/base/UIni.pas

```diff
@@ -1722,6 +1738,20 @@ procedure TIni.Load();
     Ini.JukeboxNextLineOtherOColorB := Round(Col.B);
   end;
 
+  KeysLow := IniFile.ReadString('KeyBindings', 'PianoKeysLow', '');
```

Comment:
If I'm reading this correctly, what you're doing is:
* read PianoKeysLow
* convert to array of int
* if it's length 19, use it, otherwise fall back to the default

The order in which these are happening and getting logged is a bit confusing. I can fix it fairly trivially now that config.ini is gone, just want to verify that I'm interpreting this correctly. It doesn't save anything back but I suspect it'll just save whatever it read (even if it was wrong) which is fine, and otherwise I'll make it so that it does that.

I'm going to run the CI because it will probably be angry about some end-of-line whitespace somewhere.

Reply:
The order is correct. Same for PianoKeysHigh. I'm adding the write-back to the config.ini.